pandas/core/frame.py

```diff
@@ -2359,11 +2358,11 @@ def _non_verbose_repr():
         def _sizeof_fmt(num, size_qualifier):
             # returns size in human readable format
             for x in ["bytes", "KB", "MB", "GB", "TB"]:
-                if num < 1024.0:
+                if num < 1_024.0:
                     return "{num:3.1f}{size_q} {x}".format(
                         num=num, size_q=size_qualifier, x=x
                     )
-                num /= 1024.0
+                num /= 1_024.0
```

Comment:
i dont think this adds clarity.  AFAIK we dont have a policy on this, but maybe something like 7+ digits?

Reply:
I agree, I also think for relatively small numbers this does not improve readability IMO.

Also, I don't think we need to do this for random numbers in the tests. Yes to using underscores in meaningful numbers (such as in `ccalendar.DAY_SECONDS * 1_000_000_000`), but for random values in tests asserts, I don't see the added value.